README.md

```diff
@@ -6,6 +6,15 @@ This is a profiling tool for SketchUp Ruby extensions. It wraps [`Benchmark`](ht
 
 ![Profiling in SketchUp](docs/images/profiling.png)
 
+## Installation
+There is no RBZ file! Download the ZIP archive of the code from https://github.com/SketchUp/speedup/archive/refs/heads/main.zip
```

Comment:
I would encourage to clone the repo instead of downloading it.

Though, we should start adding RBZ files for each release.